apps/site/assets/ts/stop/__tests__/DeparturesAndMapTest.tsx

```diff
@@ -62,8 +64,9 @@ describe("DeparturesAndMap", () => {
     expect(arr).toBeDefined();
   });
 
-  it("opens departure list on click", () => {
-    let departuresAndMap = render(
+  it("opens departure list on click", async () => {
+    const user = userEvent.setup();
```

Comment:
question (non-blocking): is this needed?

Reply:
everything I am reading suggests it's needed. there's a fun example [in this page](https://claritydev.net/blog/improving-react-testing-library-tests) of a utility function to run `userEvent.setup()` while rendering, but it's probably overkill :P

Reply:
It seems to have been added as a requirement with the latest major release of `testing-library`